lib/librato/rack.rb

```diff
@@ -37,6 +37,32 @@ module Librato
   #   run MyApp
   #
   class Rack
+    RECORD_RACK_IMPL = <<-'EOS'
```

Comment:
That was my guess but it wasn't totally clear, maybe `RECORD_RACK_BODY` given its use later?